app/konnect/dev-portal/applications/enable-app-reg.md

```diff
@@ -43,46 +43,39 @@ For non-`default` control planes, app registration is supported using the `konne
 
 ## Prerequisites
 
-- An API product that is versioned and published to the
-  {{site.konnect_short_name}} Dev Portal so that it appears in the catalog.
+- [An API product that is versioned and published to a {{site.konnect_short_name}} Dev Portal](/konnect/dev-portal/publish-service) so that it appears in the catalog.
 
 - If you are using [OpenID Connect](#oidc-flow) for your authorization:
 
   - Set up your application, claims, and scopes in your OpenID identity provider. Refer to your IdP/OP provider's documentation for instructions.
 
   - Edit the **Reference ID** field in the Dev Portal
     [Update Application](/konnect/dev-portal/applications/dev-apps#edit-my-app)
```

Comment:
this step is from the dev side, which doesn't make sense because the rest of the instructions (and the page) are from the admin side. I suggest removing this sentence
```suggestion
```